pkgs/games/anki/fix-paths.patch

```diff
@@ -1,99 +1,105 @@
---- anki-2.0.46/anki/lang.py.orig	2017-08-06 15:30:10.781419237 +0200
-+++ anki-2.0.46/anki/lang.py	2017-08-06 15:31:33.023043036 +0200
-@@ -71,15 +71,7 @@
-     return localTranslation().ungettext(single, plural, n)
- 
+diff --git a/anki/lang.py b/anki/lang.py
+index 44e016f..e910ed3 100644
+--- a/anki/lang.py
++++ b/anki/lang.py
+@@ -78,17 +78,7 @@ def ngettext(single, plural, n):
+     return localTranslation().ngettext(single, plural, n)
+
  def langDir():
--    dir = os.path.join(os.path.dirname(
--        os.path.abspath(__file__)), "locale")
+-    from anki.utils import isMac
+-    filedir = os.path.dirname(os.path.abspath(__file__))
+-    if isMac:
+-        dir = os.path.abspath(filedir + "/../../Resources/locale")
+-    else:
+-        dir = os.path.join(filedir, "locale")
 -    if not os.path.isdir(dir):
 -        dir = os.path.join(os.path.dirname(sys.argv[0]), "locale")
 -    if not os.path.isdir(dir):
 -        dir = "/usr/share/anki/locale"
--    if not os.path.isdir(dir):
--        dir = "/usr/local/share/anki/bin/locale"
 -    return dir
 +    return "@anki@/share/locale"
- 
+
  def setLang(lang, local=True):
      trans = gettext.translation(
-diff -Nurp anki-2.0.33.orig/anki/sound.py anki-2.0.33/anki/sound.py
---- anki-2.0.33.orig/anki/sound.py	2015-12-27 11:23:02.334908723 +0100
-+++ anki-2.0.33/anki/sound.py	2015-12-27 11:34:11.863147265 +0100
-@@ -29,8 +29,9 @@ processingDst = u"rec.mp3"
+diff --git a/anki/sound.py b/anki/sound.py
+index d8735cc..bb9e0f3 100644
+--- a/anki/sound.py
++++ b/anki/sound.py
+@@ -58,8 +58,9 @@ processingDst = "rec.mp3"
  processingChain = []
  recFiles = []
- 
+
 +lameCmd = "@lame@/bin/lame"
  processingChain = [
--    ["lame", "rec.wav", processingDst, "--noreplaygain", "--quiet"],
-+    [lameCmd, "rec.wav", processingDst, "--noreplaygain", "--quiet"],
+-    ["lame", processingSrc, processingDst, "--noreplaygain", "--quiet"],
++    [lameCmd, processingSrc, processingDst, "--noreplaygain", "--quiet"],
```

Comment:
Because we have to wrap python applications anyway, you can also add `lame` to PATH by setting `makeWrapperArgs`: https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/backup/borg/default.nix#L31

Same for mplayer. This way we reduce the number of packages.